src/services/leafletMarkersHelpers.js

```diff
@@ -101,8 +106,8 @@ angular.module('ui-leaflet').service('leafletMarkersHelpers', function ($rootSco
     };
 
     var _resetMarkerGroup = function (groupName) {
-        if (isDefined(groups[groupName])) {
-            groups.splice(groupName, 1);
+        if (angular.isDefined(groups[groupName])) {
```

Comment:
What is wrong with isDefined here? It ultimately points to the same thing with less typing.